test-cases/cdc/cdc-15m-replication.yaml

```diff
@@ -0,0 +1,38 @@
+test_duration: 3600
+
+user_prefix: 'cdc-replication'
+db_type: mixed_scylla
+
+instance_provision: 'on_demand'
+
+# 1 node initially, we start the rest from the test code with auto_bootstrap=true
+n_db_nodes: 1
+instance_type_db: 'i3.large'
+
+n_loaders: 1
+instance_type_loader: 'c4.large'
+
+n_monitor_nodes: 0
+
+nemesis_class_name: 'RandomInterruptionNetworkMonkey'
```

Comment:
May I ask why specifically this nemesis?

Reply:
From my knowledge of CDC internals I believe that CDC is a feature that can potentially be very susceptible to network partitions.

Reply:
This nemesis won't actually run unless configured in a specific way:
```
        if not self.cluster.extra_network_interface:
            raise UnsupportedNemesis("for this nemesis to work, you need to set `extra_network_interface: True`")
```
it would configure 2 network interfaces, and Scylla should be working with one of them, and the test control with a different one. FYI it a nemesis that would work only in AWS, and we run it a bit less frequently (i.e. not daily like other Nemesis) 

`RejectInterNodeNetworkMonkey` might be a bit more useful, but we disabled it cause of "reasons"

Reply:
I did enable the option:
```

# Required by the nemesis:
extra_network_interface: true 
```

What is the difference between `RandomInterruptionNetworkMonkey` and `RejectInterNodeNetworkMonkey`? Names suggest that the first is a superset of the second.

Reply:
the difference is details, i.e. the setup and tooling
`RejectInterNodeNetworkMonkey` is using iptables and works only on specific port and effect only communication between nodes,
while `RandomInterruptionNetworkMonkey` is using [tc](https://linux.die.net/man/8/tc) on a specific interface, hence effect also the clients communication.